src/onemorerep/JumpingRope.java

```diff
@@ -5,13 +5,16 @@
 
 	private double duration;
 	
-	public JumpingRope(String name, double duration) {
+	private User person;
```

Comment:
Why do we need this line if in the constructor, we're taking in User as a parameter?

Reply:
Because it is called in the caloriesBurned method.